doc/user/content/release-notes.md

```diff
@@ -54,6 +54,11 @@ Wrap your release notes at the 80 character mark.
 - Add a new metric, `mz_log_message_total` that counts the number of log
   messages emitted per severity.
 
+- Add new system tables, `mz_metrics`, `mz_metric_histograms` and
+  `mz_metrics_meta`, which import metrics exposed via prometheus once per
+  second, retaining them for 5 minutes (governed by the new
+  [command-line option `--retain-prometheus-metrics`](/cli/#introspection-sources)).
```

Comment:
This seems stray.

Reply:
That looks like something from #6271 - are you based on the correct main head?

Reply:
That is stray! Not sure how that crept in there -- probably a bad merge.